bin/rubocop-quick

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+git diff --name-status --staged | grep '^[MA]' | grep -o '\s\+.*rb' | xargs bundle exec rubocop --except Metrics --auto-correct --format quiet && \
```

Comment:
You need a file name at the end of this before the `&&`. Probably `Gemfile` is the right one.